apps/design/frontend/src/contests_screen.tsx

```diff
@@ -995,7 +995,13 @@ function EditContestForm(): JSX.Element | null {
   }
 
   const contests = listContestsQuery.data;
-  const savedContest = find(contests, (c) => c.id === contestId);
+  const savedContest = contests.find((c) => c.id === contestId);
+
+  if (!savedContest) {
+    // If the contest was just deleted, this form may still render momentarily. Ignore it.
+    return null;
+  }
```

Comment:
Thanks for fixing, this regressed due to my refactoring I believe. I'm gonna make sure there aren't other similar regressions elsewhere

Reply:
https://github.com/votingworks/vxsuite/pull/6115